### DISCUSSION TITLE
[Feature] automatic people album

### DISCUSSION THREAD

**Discussion by nopoz** [immich-app/immich#3827] [Feature Request]
### The feature

The ability to make dynamic (ie automatic) albums where you add recognized people to it rather than specific photos. When new or existing photos of that person are present in the library, they are added to the album automatically. A similar feature is available in Google Photos. Thanks!

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
You can track in #3641
